docs/csharp/programming-guide/classes-and-structs/how-to-implement-and-call-a-custom-extension-method.md

```diff
@@ -13,7 +13,7 @@ author: "BillWagner"
 ms.author: "wiwagn"
 ---
 # How to: Implement and Call a Custom Extension Method (C# Programming Guide)
-This topic shows how to implement your own extension methods for any type in the [.NET Framework Class Library](http://go.microsoft.com/fwlink/?LinkID=217856), or any other .NET type that you want to extend. Client code can use your extension methods by adding a reference to the DLL that contains them, and adding a [using](../../../csharp/language-reference/keywords/using-directive.md) directive that specifies the namespace in which the extension methods are defined.  
+This topic shows how to implement your own extension methods for any type in the [.NET Framework Class Library](https://msdn.microsoft.com/library/gg145045(v=VS.100).aspx), or any other .NET type that you want to extend. Client code can use your extension methods by adding a reference to the DLL that contains them, and adding a [using](../../../csharp/language-reference/keywords/using-directive.md) directive that specifies the namespace in which the extension methods are defined.  
```

Comment:
Perhaps we could change this sentence to get rid of that msdn link. My suggestion would be to change it to this:
This topic shows how to implement your own extension methods for any .NET type.